launch/camera.launch

```diff
@@ -3,6 +3,7 @@
     <arg name="camera_name" />
     <arg name="image_width" />
     <arg name="image_height" />
+_   <arg name="image_timestamp_offset" />
```

Comment:
Typo?

Reply:
It doesn't cause a roslaunch parse error. Could this have been doing anything nefarious?

Reply:
Idk, but I fixed it